rollup.config.js

```diff
@@ -0,0 +1,124 @@
+// rollup.config.js (building more than one bundle)
+import path                     from 'path'
+import { getBabelOutputPlugin } from '@rollup/plugin-babel';
+import alias                    from '@rollup/plugin-alias';
+import nodeResolve              from '@rollup/plugin-node-resolve'
+import json                     from '@rollup/plugin-json';
+import commonjs                 from '@rollup/plugin-commonjs';
+import dynamicImportVariables   from 'rollup-plugin-dynamic-import-variables';
+import vue                      from 'rollup-plugin-vue'
+import copy                     from 'rollup-plugin-copy'
+import { string }               from "rollup-plugin-string";
+import { terser }               from 'rollup-plugin-terser';
+import bootWebApp, { cdnHost }   from './app/boot.js';
+import injectCssToDom           from './rollup/inject-css-to-dom';
+import resolveLocalized         from './rollup/resolve-localized';
+import stripBom                 from './rollup/strip-bom';
+
+const isWatchOn = process.argv.includes('--watch');
+const outputDir = 'dist';
+
+let externals = [
+  'require', 
+  'https://cdn.cbd.int/codemirror@5.58.3/mode/xml/xml',
+  'https://cdn.cbd.int/codemirror@5.58.3/mode/handlebars/handlebars',
+  'https://cdn.cbd.int/codemirror@5.58.3/mode/htmlmixed/htmlmixed',
+  'https://cdn.cbd.int/codemirror@5.58.3/mode/javascript/javascript',
+  'https://cdn.cbd.int/codemirror@5.58.3/addon/selection/active-line',
+  'https://cdn.cbd.int/codemirror@5.58.3/addon/edit/closetag',
+  'https://cdn.cbd.int/codemirror@5.58.3/addon/edit/matchbrackets',
+  'https://cdn.cbd.int/codemirror@5.58.3/addon/comment/continuecomment.js',
+  'https://cdn.cbd.int/codemirror@5.58.3/addon/comment/comment.js',
```

Comment:
can't we use jsdeliver?

Reply:
skipping this now as the lib does not work with jsdeliver